py3-jupyterhub.yaml

```diff
@@ -42,9 +42,9 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: b4053616743eb011e5bb568700f25afca17aef53
-      repository: https://github.com/jupyterhub/jupyterhub
-      tag: ${{package.version}}
+      expected-commit: bcf5f49dd6d8b69ac21a0972755ad96ca04a252b
+      repository: xt
+      tag: ${{package.version}}b1
```

Comment:
Ignore this release it is a beta release.
add regex to ignore
https://github.com/wolfi-dev/os/blob/9d4e7b25d0abae12ccd0ea2a91a1bc806db37e35/flux-kustomize-controller.yaml#L52